torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -389,11 +412,12 @@ std::vector<Node*> getOnnxConstParentsToRemove(Node* node) {
 // known.
 void ConstantFoldONNX(Block* b, ParamMap& paramsDict, int opset_version) {
   if (opset_version != ONNX_OPSET_9 && opset_version != ONNX_OPSET_10 &&
```

Comment:
should we consider changing to `opset_version < ONNX_OPSET_9` ?

Reply:
Good point, thanks.